docs/01_cva6_user/Interfaces.rst

```diff
@@ -0,0 +1,49 @@
+..
+   Copyright (c) 2023 OpenHW Group
+   Copyright (c) 2023 Thales
+
+   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+
+.. Level 1
+   =======
+
+   Level 2
+   -------
+
+   Level 3
+   ~~~~~~~
+
+   Level 4
+   ^^^^^^^
+
+.. _cva6_interfaces:
+
+Interfaces
```

Comment:
Why not insert CVXIF in _Interfaces_ chapter ?   CVXIF chapter is big, would be helpful to dedicated a special file.

Reply:
There's more in the CV-X-IF chapter than for other interfaces, like "how to design a coprocessor", "how to program it". I realize that we lost some CV-X-IF sections when moving from GDocs to GitHub.